internal/cmd/kafka/command_broker_delete.go

```diff
@@ -47,17 +43,52 @@ func (c *brokerCommand) delete(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	promptMsg := fmt.Sprintf(errors.DeleteResourceConfirmYesNoMsg, "broker", brokerIdStr)
-	if ok, err := form.ConfirmDeletion(cmd, promptMsg, ""); err != nil || !ok {
+	brokerIdToNumId := make(map[string]int32)
+	if err := c.confirmDeletion(cmd, restClient, restContext, clusterId, args, brokerIdToNumId); err != nil {
 		return err
 	}
 
+	var errs error
+	var deleted []string
 	opts := kafkarestv3.ClustersClusterIdBrokersBrokerIdDeleteOpts{ShouldShutdown: optional.NewBool(true)}
-	_, resp, err := restClient.BrokerV3Api.ClustersClusterIdBrokersBrokerIdDelete(restContext, clusterId, brokerId, &opts)
-	if err != nil {
-		return kafkarest.NewError(restClient.GetConfig().BasePath, err, resp)
+	for _, id := range args {
+		if _, resp, err := restClient.BrokerV3Api.ClustersClusterIdBrokersBrokerIdDelete(restContext, clusterId, brokerIdToNumId[id], &opts); err != nil {
+			errs = errors.Join(errs, kafkarest.NewError(restClient.GetConfig().BasePath, err, resp))
+		} else {
+			deleted = append(deleted, id)
+		}
+	}
+	if len(deleted) == 1 {
+		output.Printf("Started deletion of broker %[1]s. To monitor the remove-broker task run `confluent kafka broker get-tasks %[1]s --task-type remove-broker`.\n", deleted[0])
+	} else if len(deleted) > 1 {
+		output.Printf("Started deletion of brokers %s. To monitor a remove-broker task run `confluent kafka broker get-tasks <id> --task-type remove-broker`.\n", utils.ArrayToCommaDelimitedString(deleted, "and"))
+	}
```

Comment:
refactor this like other delete's, to be consistent?

Reply:
I dunno... I kept it this way because this one had a special output to begin with, and I don't want to modify the common delete message function for one special case.